modules/mobile/docs/openapi.yaml

```diff
@@ -1361,7 +1517,7 @@ components:
               example: VA
             status:
               type: string
-              enum: [BOOKED, CANCELLED]
+              enum: [ BOOKED, CANCELLED ]
```

Comment:
looks like our IDEs have different settings for array formatting

Reply:
should be fixed now